modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/services/liferay/object/mdf-requests/createMDFRequest.ts

```diff
@@ -10,21 +10,69 @@
  */
 
 import {Liferay} from '../..';
+import GetTotalBudget from '../../../../../routes/MDFRequestForm/steps/Review/utils/GetTotalBudget';
 import MDFRequest from '../../../../interfaces/mdfRequest';
+import MDFRequestActivity from '../../../../interfaces/mdfRequestActivity';
 import {LiferayAPIs} from '../../common/enums/apis';
 import liferayFetcher from '../../common/utils/fetcher';
 
+interface DateActivities {
+	endDates: Date[];
+	startDates: Date[];
+}
+
+const getDateActivities = (mdfRequest: MDFRequest) => {
```

Comment:
@joaosvassis, should be good add this function in utils directory with the interface too.